gammapy/data/data_store.py

```diff
@@ -662,12 +662,16 @@ def make_psf(self, position, energy=None, theta=None):
 
         offset = position.separation(self.pointing_radec)
         if not energy:
-            energy = self.psf.to_table_psf(theta=offset).energy
+            energy = self.psf.to_energy_dependent_table_psf(theta=offset).energy
         if not theta:
-            theta = self.psf.to_table_psf(theta=offset).offset
+            theta = self.psf.to_energy_dependent_table_psf(theta=offset).offset
 
-        psf_value = self.psf.to_table_psf(theta=offset, offset=theta)\
-            .evaluate(energy)
+        if (type(self.psf) == PSF3D):
+            psf_value = self.psf.to_energy_dependent_table_psf(theta=offset) \
+                .evaluate(energy)
+        else:
+            psf_value = self.psf.to_energy_dependent_table_psf(theta=offset, offset=theta) \
```

Comment:
This is crazy: `theta=offset, offset=theta`
Things should be named consistently.

It's not a comment to you to address here.
You could file a reminder issue or leave a `TODO: make axis names consistent` comment here.